spec/models/activity_spec.rb

```diff
@@ -1402,4 +1402,34 @@
       expect(fund.is_newton_funded?).to be_falsey
     end
   end
+
+  describe "#status" do
```

Comment:
Aren't these already covered by `ProgrammeToIatiStatus` tests?

Reply:
Yep, but this was when I wasn't sure we'd be keeping `ProgrammeToIatiStatus`, pending my questions to Meyric. As it is, I think I'll wait until #843 is merged, then use the new `load_yaml` method to do that, possibly in the presenter like you suggested, and then remove the service object after all.

Reply:
Good call! The YAML files are the right place to store these BEIS-approved mappings 🎉